source/imt-terms.ttl

```diff
@@ -52,11 +57,26 @@
     skos:inScheme </term/imt> ;
     skos:exactMatch iana:text\/turtle .
 
+<text/csv> a :EncodingFormat ;
+    skos:notation "text/csv" ;
+    skos:inScheme </term/imt> ;
+    skos:exactMatch iana:text\/csv .
+
+<text/tab-separated-values> a :EncodingFormat ;
+    skos:notation "text/tab-separated-values" ;
+    skos:inScheme </term/imt> ;
+    skos:exactMatch iana:text\/tab-separated-values .
+
 <text/xml> a :EncodingFormat ;
     skos:notation "text/xml" ;
     skos:inScheme </term/imt> ;
     skos:exactMatch iana:text\/xml .
 
+<video/JPEG> a :EncodingFormat ;
+    skos:notation "video/JPEG" ;
+    skos:inScheme </term/imt> ;
+    skos:exactMatch iana:video\/JPEG .
+
```

Comment:
Är det någon tanke att lägga till även ```<application/vnd.openxmlformats-officedocument.spreadsheetml.sheet> a :EncodingFormat ;
    skos:notation "application/vnd.openxmlformats-officedocument.spreadsheetml.sheet" ;
    skos:inScheme </term/imt> ;
    skos:exactMatch iana:application\/application\/vnd.openxmlformats-officedocument.spreadsheetml.sheet .
```
och
``` <application/msword> a :EncodingFormat ;        
    skos:notation "application/msword" ;        
    skos:inScheme </term/imt> ;                 
    skos:exactMatch iana:application\/msword . ```
    
    För excel arken som följer med ljudfilerna (om den gör det) och doc filerna som följer med Post- och inrikes tidningar tror jag att det var?

Reply:
Bra fråga. Jag funderade på dom oxå men blev osäker på om excel ska med för ljudfilerna iom att den ju bara innehåller info om vilka titlar varje fil har och SMDB-urlar? Tänker att den infon kommer vi ju ha i Libris? Men inte helt säker.
Likadant med `.doc` för Post- och inrikestidningar. Osäker om de ska in direkt nu i början? Men å andra sidan så är det ju ändå inte fel att lägga till excel och doc.